security/src/main/java/com/devonfw/tools/security/UrlFileFilter.java

```diff
@@ -0,0 +1,22 @@
+package com.devonfw.tools.security;
+
+import com.devonfw.tools.ide.url.model.file.UrlStatusFile;
+import com.devonfw.tools.ide.url.updater.UpdateManager;
+
+import java.io.FileFilter;
+import java.nio.file.Paths;
+
+/**
+ * This {@link FileFilter} only accepts files with name {@link UrlStatusFile#STATUS_JSON}. The paths of the accepted
+ * files are then analyzed by the {@link UrlAnalyzer} to detect the available tools, editions and versions.
+ */
+public class UrlFileFilter implements FileFilter {
+
+  UpdateManager updateManager = new UpdateManager(Paths.get("C:\\projects\\_ide\\urls"), null);
```

Comment:
Not sure about that, seems to be a default. Was it merged?

Reply:
oh. i missed this. thanks